build-tools/settings.gradle

```diff
@@ -1,19 +1,21 @@
+pluginManagement {
+    apply from: "plugin-management.gradle", to: pluginManagement
+}
+
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version("0.4.0")
```

Comment:
It's unrelated to these changes, but I see this plugin is also used inside the [settings.gradle](
https://github.com/androiddevelopersitalia/aditogether/pull/68/files#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30deR6). Could be useful to track it inside the `libs.version.toml`?

Reply:
Unfortunately we can't do it, it's a limitation of Gradle. Plugins applied in `settings.gradle` can't access version catalogs.
I added the issue https://github.com/androiddevelopersitalia/aditogether/issues/65 few days ago to find a way to share the settings plugins between multiple Gradle projects

Reply:
Looks good tnx!